go.mod

```diff
@@ -65,12 +64,24 @@ require (
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/yvasiyarov/go-metrics v0.0.0-20140926110328-57bccd1ccd43 // indirect
 	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20140908184405-b21fdbd4370f // indirect
-	go.opencensus.io v0.22.4 // indirect
+	go.opencensus.io v0.24.0 // indirect
 	golang.org/x/net v0.7.0 // indirect; updated for CVE-2022-27664, CVE-2022-41717
 	golang.org/x/sys v0.5.0 // indirect
 	golang.org/x/text v0.7.0 // indirect
-	google.golang.org/appengine v1.6.6 // indirect
-	google.golang.org/genproto v0.0.0-20200825200019-8632dd797987 // indirect
-	google.golang.org/grpc v1.31.0 // indirect
-	google.golang.org/protobuf v1.26.0 // indirect
+	google.golang.org/appengine v1.6.7 // indirect
+	google.golang.org/genproto v0.0.0-20230110181048-76db0878b65f // indirect
+	google.golang.org/grpc v1.51.0 // indirect
+	google.golang.org/protobuf v1.28.1 // indirect
+)
+
+require cloud.google.com/go/storage v1.29.0
+
+require (
+	cloud.google.com/go/compute v1.14.0 // indirect
+	cloud.google.com/go/compute/metadata v0.2.3 // indirect
+	cloud.google.com/go/iam v0.8.0 // indirect
+	github.com/google/go-cmp v0.5.9 // indirect
+	github.com/google/uuid v1.3.0 // indirect
+	github.com/googleapis/enterprise-certificate-proxy v0.2.1 // indirect
+	golang.org/x/xerrors v0.0.0-20220907171357-04be3eba64a2 // indirect
 )
```

Comment:
Looks like these landed in separate groups; can you move them into the first (direct dependencies) and second (indirect dependencies) groups in the `go.mod`?

Reply:
@jmontleon what @thaJeztah means is, `go.mod` now has several `require` sections and even some `require` on individual line -- I suspect this is due to either `gopls` being silly (happened to me) or the IDE you use is causing this. It'd be good to group the direct and indirect deps into their dedicate `require` stanzas.

Reply:
Yep, I can go ahead and do this and squash all the commits. Like I mentioned early on I mostly just separated them so it was a little easier to review the actual code changes go mod updates. (https://github.com/distribution/distribution/pull/3702#issuecomment-1199766143)